src/string-class.js

```diff
@@ -0,0 +1,165 @@
+/* @file string-class */
+const stringClassExtensions = {
+  /**
+   * hasVowels
+   * Test to see if a string contains vowels
+   * @return {Boolean} true or false
+   */
+  hasVowels() {
+    const vowels = /[aeiou]/gi;
```

Comment:
Nitpick: It may probably be better to return the match directly instead of declaring a whole new variable for that purpose. Don't you think so? (like: `return /[aeiou]/gi.test(this)`